### PR TITLE
[WIP] 9.0 migration of hr_holidays_legal_leave

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by feketemihai** [comment]
@damdam-s @gurneyalex Thank for update the module, can i ask to update readme, since the HR config isn't there anymore with the text from version 10.

Configuration

To configure this module, you need to:

Company setup \* Go to the company and select the leave type to use as legal leave

with 

Configuration

To configure this module, you need to:

assign legal leave type via Leaves > Configuration:
choose Leave type you need
click [Edit]
set Legal/Annual checkbox
click [Save]

**Comment by damdam-s** [comment]
@feketemihai no problem. I do it right now

**Comment by feketemihai** [comment]
:+1:

File: hr_holidays_legal_leave/models/hr_holidays_status.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models, api
+
+
+class HolidaysType(models.Model):
+    _inherit = "hr.holidays.status"
+
+    is_annual = fields.Boolean(
```
Comment (pedrobaeza) [inline hr_holidays_legal_leave/models/hr_holidays_status.py:9]: Although I know this is what is now in v10, I'm seeing that this is not comfortable for user with access to several companies. We should have a `company_id` field that is filled by default with user company, but with the possibility to be switched. This is similar to Accounting settings.

Comment (damdam-s) [reply hr_holidays_legal_leave/models/hr_holidays_status.py:9]: @pedrobaeza you're right (I think) but is it correct to have a different behaviour between 9 and 10 versions ?
If you want me to do it, I can but I will make a PR on v10 too to have the same behaviour.

@gurneyalex @feketemihai cc

Comment (pedrobaeza) [reply hr_holidays_legal_leave/models/hr_holidays_status.py:9]: Yes, please. I think it's a good change. We can cherry-pick the commit directly in v10 when done in v9.

Comment (damdam-s) [reply hr_holidays_legal_leave/models/hr_holidays_status.py:9]: @pedrobaeza done in  df9830f. Is it what you wanted ?

**Review by pedrobaeza — APPROVED** [review]
Yeah, thanks

**Comment by damdam-s** [comment]
great! thanks

**Comment by pedrobaeza** [comment]
Ouch, it lacks the _company_id_ field in the view. I'll add it directly

**Comment by damdam-s** [comment]
aww sorry for that :(

**Comment by pedrobaeza** [comment]
I have include both commits in v10, cc @yelizariev

**Pull request merged**